module/zfs/spa.c

```diff
@@ -3408,6 +3415,14 @@ spa_activity_check(spa_t *spa, uberblock_t *ub, nvlist_t *config)
 			break;
 		}
 		error = 0;
+
+		if (interations++ % 30 == 0 && import_expire > gethrtime()) {
+			uint64_t remaining = import_expire - gethrtime();
```

Comment:
My first thought was that gethrtime() may actually cost something on older hardware, but the second is that it may change in time and you may get negative value here.

Reply:
Thanks.  Fixed